### PR TITLE
[HNC-571] - Adding Frogbot CLI to gh-compoiste-actions.

### PR DESCRIPTION
**Pull request by mayur-hitachivantara**

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -438,16 +438,22 @@ runs:
...
+      if: ${{ env.JF_URL }}
+      run: /opt/frogbot scan-pull-request
+      shell: bash
       env:
         JF_URL: ${{ env.JF_URL }}
         JF_ACCESS_TOKEN: ${{ env.JF_ACCESS_TOKEN }}
         JF_GIT_TOKEN: ${{ env.JF_GIT_TOKEN }}
+        JF_GIT_PROVIDER: 'github'
+        JF_PATH_EXCLUSIONS: ${{ env.JF_PATH_EXCLUSIONS }}
+        JF_GIT_OWNER: ${{ env.GIT_REPO_OWNER}}
```
Comment (cardosov) [inline action.yaml:451]: Where are these values coming? Can't they come from Github's context?

File: action.yaml [hunk 2]
```diff
@@ -438,16 +438,22 @@ runs:
         cat .frogbot/frogbot-config.yml
```
Comment (cardosov) [inline action.yaml:438]: Do we still need this configuration file?

File: action.yaml [hunk 3]
```diff
@@ -438,16 +438,22 @@ runs:
...
         JF_GIT_TOKEN: ${{ env.JF_GIT_TOKEN }}
+        JF_GIT_PROVIDER: 'github'
+        JF_PATH_EXCLUSIONS: ${{ env.JF_PATH_EXCLUSIONS }}
+        JF_GIT_OWNER: ${{ env.GIT_REPO_OWNER}}
+        JF_GIT_REPO: ${{ env.GIT_REPO_NAME }}
+        JF_GIT_PULL_REQUEST_ID : ${{ env.PULL_REQUEST_NUMBER }}
         JF_WORKING_DIR: ${{ env.JF_WORKING_DIR }}
-        JF_FAIL: "FALSE"
-
+        JF_FAIL: ${{ env.JF_FAIL }}
```
Comment (cardosov) [inline action.yaml:455]: Why not `${{ env.JF_FAIL || 'TRUE' }}`?

**Review by cardosov — CHANGES_REQUESTED** [review]
Don't know if you ever will be able to handle these questions but, at least, they will have visibility in this PR.

File: action.yaml (hunk 1 above)
Comment (mayur-hitachivantara) [reply action.yaml:451]: These values will be passed by the user while calling the frogbot step of composite action as shown [here](https://github.com/pentaho/actions-common/blob/f7ca99a69427a1192991e76b956d9d6da318256e/.github/workflows/pr.yml#L150).

File: action.yaml (hunk 2 above)
Comment (mayur-hitachivantara) [reply action.yaml:438]: No We don't need this file

File: action.yaml (hunk 3 above)
Comment (mayur-hitachivantara) [reply action.yaml:455]: I will try with this one

File: action.yaml (hunk 2 above)
Comment (cardosov) [reply action.yaml:438]: So this step is going to be removed?

File: action.yaml [hunk 4]
```diff
@@ -438,16 +438,22 @@ runs:
...
+        JF_GIT_PROVIDER: 'github'
+        JF_PATH_EXCLUSIONS: ${{ env.JF_PATH_EXCLUSIONS }}
+        JF_GIT_OWNER: ${{ env.GIT_REPO_OWNER}}
+        JF_GIT_REPO: ${{ env.GIT_REPO_NAME }}
+        JF_GIT_PULL_REQUEST_ID : ${{ env.PULL_REQUEST_NUMBER }}
         JF_WORKING_DIR: ${{ env.JF_WORKING_DIR }}
-        JF_FAIL: "FALSE"
-
+        JF_FAIL: ${{ env.JF_FAIL || 'TRUE' }}
+
```
Comment (cardosov) [inline action.yaml:456]: Unneeded white-spaces

**Review by cardosov — CHANGES_REQUESTED** [review]

**Review by AdityaHitachi — APPROVED** [review]

**Pull request merged**